client/html/templates/common/summary/detail-body-default.php

```diff
@@ -201,6 +201,20 @@
 						<a class="product-name" href="<?php echo $enc->attr( $this->url( $detailTarget, $detailController, $detailAction, $params, array(), $detailConfig ) ); ?>">
 <?php		echo $enc->html( $product->getName(), $enc::TRUST ); ?>
 						</a>
+
+						<p class="code">
+							<span class="name">
+							<?php
+								echo $enc->html( $this->translate('client/html', 'Article no.:'), $enc::TRUST);
```

Comment:
The translation domain is 'client' only for the latest version. 'client/html' was used up to the last LTS version